Documentation/install.rst

```diff
@@ -1097,30 +1097,41 @@ If for some reason you do not want to run Cilium as a container image.
 Installing it from source is possible as well. It does come with additional
 dependencies described in :ref:`admin_system_reqs`.
 
+0. Download and install go-bindata
+
+    $ go get -u github.com/jteeuwen/go-bindata/...
+
 1. Download & extract the latest Cilium release from the ReleasesPage_
 
 .. _ReleasesPage: https://github.com/cilium/cilium/releases
 
 .. code:: bash
 
-    $ wget https://github.com/cilium/cilium/archive/v0.10.0.tar.gz
-    $ tar xzvf v0.10.0.tar.gz
-    $ cd cilium-0.10.0
+   $ go get github.com/cilium/cilium
+   $ cd $GOPATH/src/github.com/cilium/cilium
 
 2. Build & install the Cilium binaries to ``bindir``
 
 .. code:: bash
 
-   $ make
+   $ git checkout v0.11
```

Comment:
cc: @scanf 
Is there a way we can have this be updated with a variable that points to the latest v.{latest version} tag? I want to reduce the amount of documentation updates we have to do after we release each version if possible.

Reply:
So this we can do, but not for the code block. Need to figure out how to get a consistent look(font and frame) between `parsed-literal` and `code`.

```diff
diff --git a/Documentation/conf.py b/Documentation/conf.py
index fc907438693d..83a6e5df509d 100644
--- a/Documentation/conf.py
+++ b/Documentation/conf.py
@@ -57,7 +57,7 @@ author = u'Cilium Authors'
 # built documents.
 #
 # The short X.Y version.
-release = re.sub('^v', '', os.popen('git describe --tags').read().strip())
+release = re.sub('^v', '', os.popen('git describe --tags $(git rev-list --tags --max-count=1)').read().strip())
 # The full version, including alpha/beta/rc tags.
 version = release

@@ -69,7 +69,9 @@ scm_web = githubusercontent + branch
 # Store variables in the epilogue so they are globally available.
 rst_epilog = """
 .. |SCM_WEB| replace:: \{s}
-""".format(s = scm_web)
+
+.. |VERSION| replace:: \{v}
+""".format(s = scm_web, v = version)

 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
diff --git a/Documentation/install.rst b/Documentation/install.rst
index 19852de0ec8d..51e43e2d7777 100644
--- a/Documentation/install.rst
+++ b/Documentation/install.rst
@@ -1112,9 +1112,9 @@ dependencies described in :ref:`admin_system_reqs`.

 2. Build & install the Cilium binaries to ``bindir``

-.. code:: bash
+.. parsed-literal::

-   $ git checkout v0.11
+   $ git checkout \ |VERSION|\/
    $ # We are pointing to $GOPATH/bin as well since it's where go-bindata is
    $ # installed
    $ PATH=$GOPATH/bin:$PATH make
```